knowledge-base/chart-datatable-binding.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to use DataTable as source of data for the Chart
+description: How to convert DataTable to List and use it as data source for the Chart
+type: how-to
+page_title: How to use DataTable as data source for the Chart
+slug: chart-datatable-binding
+position: 
+tags: 
+ticketid: 1589460 
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Charts for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I want to use DataTable as a data source for the Chart series.
+
+
+## Solution
+
+The Telerik UI for Blazor Chart does not support DataTable binding out of the box. To use DataTable instance as a data source for the series of the Chart you have to convert the DataTable to IEnumerable<T> e.g. List<T>.
+
+Here is a simple example:
+
+````CSHTML
+@using System.Data;
+@using System.Dynamic;
+
+<TelerikChart>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Pie"
+                     Data="Data"
```

Comment:
We use @ in such places. See our docs and demos code style guide in the `blazor` repo wiki.

```suggestion
                     Data="@Data"
```